tests/components/playstation_network/snapshots/test_diagnostics.ambr

```diff
@@ -88,5 +86,22 @@
       }),
       'username': '**REDACTED**',
     }),
+    'groups': dict({
+      'test-groupid': dict({
+        'groupName': dict({
+          'value': '',
+        }),
+        'members': list([
+          dict({
+            'accountId': '**REDACTED**',
+            'onlineId': '**REDACTED**',
+          }),
+          dict({
+            'accountId': '**REDACTED**',
+            'onlineId': '**REDACTED**',
+          }),
+        ]),
```

Comment:
I would propose to either omit the members list, or to just add the length in here if that is relevant.